python/graphscope/framework/graph.py

```diff
@@ -619,3 +619,37 @@ def deserialize(cls, path, sess, **kwargs):
         return cls(
             sess.session_id, VineyardObject(object_id=int(vineyard.ObjectID(graph_id)))
         )
+
+    def draw(self, vertices, hop=1):
+        """Visualize the graph data in the result cell when the draw functions are invoked
+
+        Args:
+            vertices (list): selected vertices.
+            hop (int): draw induced subgraph with hop extension. Defaults to 1.
+
+        Returns:
+            A GraphModel.
+        """
+        from ipygraphin import GraphModel
+
+        sess = get_session_by_id(self.session_id)
+        interactive_instance_dict = sess.interactive_instance()
+        if (
+            self._vineyard_id in interactive_instance_dict
+            and interactive_instance_dict[self._vineyard_id] is not None
+        ):
+            interactive_query = interactive_instance_dict[self._vineyard_id]
+        else:
+            interactive_query = sess.gremlin(self)
+
+        if interactive_query is None:
+            raise ValueError(
+                "Failed to obtain interactive instance, unable to query data and draw graph."
+            )
```

Comment:
I think `sess.gremlin(self)` should be enough since in `session.gremlin()` there are already a cache behaviour.